components/module.py

```diff
@@ -0,0 +1,27 @@
+class Module:
+    # instances
+    # module_number = 0
+
+    # **kwargs for json
+    def __init__(self, module_number, **kwargs):
+        self.module_number = module_number
+        module_number += 1
+        self.scenes = []
+        self.chars_on_screen = []
```

Comment:
let's name this `present_characters`, so we don't mix up chars ('a', 'b') and characters (like Selma)